### ISSUE TITLE
A11y_VSCode_ChangingtheUsersettings_ScreenReader: Screen reader does not announce the ordering of items present in the dropdown list of 'Editor: Accessibility Support' combo box.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#76865]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version : MacOS Mojave 10.14.5

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : Voice over

**Repro Steps:**
1.Open Visual studio code editor.
2.Open the command Pallet by pressing "F1" key and type "setting".
3.Using down-arrow key select user setting.
4.Type the word "Auto " and select one of the suggestion given by the vs code editor.
5.Type the "Tab size" and select the option from the auto suggestion.
6.Start screen reader

**Actual:**
Screen reader does not announce the ordering that is 'auto 1 of 3' for drop down item of 'Editor: Accessibility Support' combo box.

This issue is observed in Mac + Voice over

**Expected:**
Screen reader should announce the proper ordering that is 'auto 1 of 3' for each drop down item of 'Editor: Accessibility Support' combo box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not get the exact information.

**MAS Reference:**
**MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference:
[A11y_VSCode_ChangingtheUsersettings_ScreenReader.pptx](https://github.com/microsoft/vscode/files/3367937/A11y_VSCode_ChangingtheUsersettings_ScreenReader.pptx)

**Comment by isidorn** [comment]
That is the native HTML select box. Just how it behaves.
Please file issue against Voice Over that they do not read the item order in the HTML native select box.
